ui/app/pages/routes/index.js

```diff
@@ -192,47 +193,50 @@ class Routes extends Component {
     const { transaction: sidebarTransaction } = props || {}
 
     return (
-      <div
```

Comment:
Can we get rid of this patching by having another component used instead of `Routes` and then inside of it, it renders `Routes`?

Reply:
done